src/Styles/Genre.scss

```diff
@@ -1,6 +1,8 @@
 .genre-container {
   border: 1px solid black;
   margin: 40px;
+  width: 1350px;
```

Comment:
I would encourage working with `rems` or `ems` over `px`. `px` are bad for responsive design and accessibility.